bundler/helpers/v2/spec/functions/dependency_source_spec.rb

```diff
@@ -115,6 +115,9 @@
         stub_request(:get, registry_url + "specs.4.8.gz").
           with(basic_auth: ["SECRET_CODES", ""]).
           to_return(status: 400)
+        stub_request(:get, registry_url + "info/business").
```

Comment:
For the sake of posterity (aka future me 😅), could you give some context on why this stub is needed?